node/actors/bft/src/testonly/ut_harness.rs

```diff
@@ -406,19 +426,11 @@ impl UTHarness {
     }
 
     pub(crate) async fn sim_timeout(&mut self) {
-        scope::run!(&self.ctx, |ctx, s| {
```

Comment:
ditto for other calls you have wrapped in scope::run

Reply:
As discussed it will be cleaned in a follow-up PR.